src/main/java/training/busboard/Models/Bus.java

```diff
@@ -0,0 +1,40 @@
+package training.busboard.Models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+public class Bus {
+
+    public int timeToStation;

```

Comment:
When you have a fields containing amounts, it's useful to include the the units in the name, e.g. `secondsToStation` or `timeToStationInSeconds`. It might save someone from misinterpreting it and introducing a bug.

Even better would be to add some type safety by storing it as a [Duration](https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html).